controller/api/views.py

```diff
@@ -385,3 +385,13 @@ def destroy(self, request, **kwargs):
         user.is_superuser = user.is_staff = False
         user.save(update_fields=['is_superuser', 'is_staff'])
         return Response(status=status.HTTP_204_NO_CONTENT)
+
+
+class UserView(BaseDeisViewSet):
+    """A Viewset for interacting with User objects."""
+    model = User
+    serializer_class = serializers.UserSerializer
+    permission_classes = [permissions.IsAdmin]
+
+    def get_queryset(self):
+        return self.model.objects.all()
```

Comment:
This should be changed to
```python
from django.contrib.auth.models import AnonymousUser
...
    def get_queryset(self):
        return self.model.objects.exclude(username='AnonymousUser')
```
Otherwise we get the psuedo-user `AnonymousUser` in the listing. which is unintended. 
```console
$ deis users:list
=== Users
AnonymousUser
test
```
And the unit tests should be changed to match.